build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+set -e
+
+#/ build.sh [--qemu]
+#/
+#/ Leverages docker to produce a Print Nanny image, optionally targeting qemu
+#/ (untested) instead of the Raspberry Pi 4.
+
+if ! command -v docker >/dev/null 2>&1; then
+  echo "⛔ docker is required for build.sh, but it could not be found. please install docker and try again." >&2
+  exit 1
+fi
+
+usage() {
+  grep '^#/' "$0" | cut -c 4- >&2
+  exit 1
+}
+
+MACHINE="raspberrypi4-64"
+while [ "$#" -gt 0 ]; do
+  case "$1" in
+    --qemu) MACHINE=qemux86-64; shift ;;
+    *) usage ;;
+  esac
+done
+
+DOCKER_BUILDKIT=true docker build -t printnanny-yocto -f docker/Dockerfile .
+
+docker run \
+  --mount type=volume,source=printnanny-yocto-cache,destination=/poky/build/cache \
```

Comment:
You'll probably want to mount `/poky/build/tmp/deploy/` too - this is where build artifacts go (images, RPM)

Reply:
Hrm and `/poky/build/downloads` as well. Maybe mounting the entire build dir is best for now. You might run into strange behavior with the incremental cache if you're only mounting certain paths.

Reply:
Updated to include `tmp`; `deploy` is relocated via a `local.conf` change, and the new path is managed as a `bind` volume (thus outputs are written to the host filesystem and not a docker volume).